src/ToolBox/SOS/Strike/sos.cpp

```diff
@@ -150,7 +150,7 @@ namespace sos
             
         
         if (mTypeName == NULL)
-            return L"<error>";
+            return (wchar_t *)W("<error>");
```

Comment:
It may be to change the return type to const wchar_t * instead of adding this cast.